tests/config/test_config_classes.py

```diff
@@ -1,10 +1,24 @@
+import os
+import shutil
+
 from glambie.config.config_classes import GlambieRunConfig
+from glambie.config.config_classes import RegionRunConfig
+from glambie.const.constants import YearType
+from glambie.const.data_groups import GlambieDataGroup
+from glambie.processing.path_handling import OutputPathHandler
 import pytest
-import os
 import yaml
-from glambie.const.constants import YearType
 
-from glambie.const.data_groups import GlambieDataGroup
+TESTING_DIR = os.path.join("tests", "test_config_outputs")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def output_path_handler():
+    def remove_test_dir():
```

Comment:
please use `tempfile.TemporaryDirectory` as a context manager to do this.

Reply:
Hmmmm, no that's not correct as this has no teardown

Reply:
[tmp_path](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) is not simply an argument, it's a built in fixture that is torn down correctly at the end of every test. It's the default way to create a temporary path for a single test.

Reply:
```
@pytest.fixture(scope="session", autouse=True)
def output_path_handler(tmp_path: os.PathLike):
    yield OutputPathHandler(tmp_path)
```
doesn't run. And in fact flake8 flags it up as "no teardown in fixture output_path_handler2, use return instead of yield"

Reply:
yes, flake8 is pointing out that yield is a bit pointless now (because the teardown is now fully implied by the use of tmp_path) and return is the correct thing to use.

Reply:
Changed now to use tmp_path in both occurences.